pfio/v2/__init__.py

```diff
@@ -0,0 +1,36 @@
+'''
+fs.FS> interface
+implementations:
+open_fs(URI, container=None|zip) => Local/HDFS/S3/Zip, etc
+- Local
+  - subfs() -> Local
+  - open_zip() -> Zip
```

Comment:
By using this name, are we not going to support containers other than zip

Reply:
Do we have any other container format in the scope? I would like being as specific as we need.